rusk/src/bin/config/chain.rs

```diff
@@ -62,4 +64,8 @@ impl ChainConfig {
     pub(crate) fn migration_height(&self) -> Option<u64> {
         self.migration_height
     }
+
+    pub(crate) fn generation_timeout(&self) -> Option<Duration> {
```

Comment:
Could you add it to `default.config.toml` together with a short explanation how to use this config - recommended values etc.

Reply:
efc9717e47e34f25e13116021baace688303f2dd